.eslintrc.js

```diff
@@ -202,54 +202,69 @@ module.exports = {
     },
     {
       files: ["site/**"],
+      parserOptions: {
+        tsconfigRootDir: `${__dirname}/site`,
+        project: "tsconfig.json",
+      },
```

Comment:
Bonus: I removed `site/.eslintrc.json` that only contained this block. We now how all `overrides` in one place, which makes ESLint rules easier to manage in the monorepo of our size.